test/parser/test_captcha_hit.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+from harambe.errors import CaptchaError
+
+
+def simulate_captcha_scenario():
+    # Simulating a CAPTCHA hit scenario
+    raise CaptchaError()
+
+
+def test_captcha_hit_exception():
+    # Testing if CaptchaHitException is raised when expected
+    with pytest.raises(CaptchaError, match="CAPTCHA was hit."):
+        simulate_captcha_scenario()
```

Comment:
think we can delete this test.

If they extend a single parent error, we can test that it raises the ParentError maybe. Otherwise not really worthwhile

Reply:
`python
import pytest
from harambe.errors import CaptchaError, HarambeException


def simulate_captcha_scenario():
    # Simulating a CAPTCHA hit scenario
    raise CaptchaError()


def test_captcha_hit_exception():
    # Testing if CaptchaHitException is raised when expected
    with pytest.raises(HarambeException, match="CAPTCHA was hit."):
        simulate_captcha_scenario()
`

Reply:
okay @asim-shrestha ?

Reply:
i am verifying it raises the parent exception